tests/data_validation/data_validation_generics_side_panel_component.test.ts

```diff
@@ -18,7 +18,7 @@ mockGetBoundingClientRect({
   "o-dv-type": () => dataValidationSelectBoundingRect,
 });
 
-export async function mountDataValidationPanel(model?: Model) {
+async function mountDataValidationPanel(model?: Model) {
```

Comment:
remove this and use the new helper

Reply:
What is 'the new helper' ?

Reply:
Oh you duplicated the helper in both test files, I though you defined a new helper that you could import everywhere. 

Maybe you should define it somewhere so you can import it from both files ? But probably we don't care, it's only like 4 lines.